mongo/change_stream.go

```diff
@@ -132,7 +132,7 @@ func newChangeStream(ctx context.Context, config changeStreamConfig, pipeline in
 	cs.aggregate = operation.NewAggregate(nil).
 		ReadPreference(config.readPreference).ReadConcern(config.readConcern).
 		Deployment(cs.client.deployment).ClusterClock(cs.client.clock).
-		CommandMonitor(cs.client.monitor).Session(cs.sess).ServerSelector(cs.selector).Retry(driver.RetryNone).
+		CommandMonitor(cs.client.monitor).Session(cs.sess).ServerSelector(cs.selector).Retry(driver.RetryChangeStream).
```

Comment:
Would operation execution interpret `driver.RetryChangeStream` the same as `driver.RetryNone`? If [this](https://github.com/mongodb/mongo-go-driver/blob/master/x/mongo/driver/operation.go#L737) branch is ever executed under the previous command monitor conditions, then I don't believe it would any longer with these changes.

Reply:
I've removed the `RetryChangeStream` `RetryMode`, so I believe this is irrelevant now. Good point, though!